database_setup/akka-persistence-postgres.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE IF NOT EXISTS public.journal (
   deleted BOOLEAN DEFAULT FALSE,
   tags VARCHAR(255) DEFAULT NULL,
   message BYTEA NOT NULL,
+  from_migration BOOLEAN DEFAULT FALSE,
```

Comment:
We probably don't want this column. If you've found it in prod or preprod then we should drop it from those database tables. It was only used when we migrated to ACP several years ago

Reply:
I added it because while copying eventstore from prod it doesnt work without it . May be need to delete in prod .

Reply:
Yes that's what I mean - we can drop the column in the prod database. I think it's already been dropped from preprod